app/controllers/article_controller.rb

```diff
@@ -39,7 +39,11 @@ def ignore_request_unless_permitted
     return if permitted?
 
     flash_notice(:permission_denied.t)
+    # rubocop disable Style/AndOr
+    # RuboCop 0.83 autocorrects the following line to:
+    #   redirect_to(action: "index_article") && (return)
     redirect_to(action: "index_article") and return
+    # rubocop enable Style/AndOr
```

Comment:
I've been through this one and the short story is that this should be come just:
```
redirect_to(action: "index_article")
return
```